src/services/activityReports.js

```diff
@@ -228,7 +228,80 @@ export function activityReports() {
       },
       {
         model: User,
-        attributes: ['name', 'role', 'fullName'],
+        attributes: ['id', 'name', 'role', 'fullName'],
+        as: 'collaborators',
+        through: { attributes: [] },
+      },
+    ],
+  });
+}
+/**
+ * Retrieves alerts based on the following logic:
+ * One or both of these high level conditions are true -
+ * manager - approvingManagerId matches and report status is submitted.
+ * specialist - author id or one of the collaborator's id matches and status is not approved nor
+ * submitted.
+ * @param {*} userId
+ */
+export function activityReportAlerts(userId) {
+  return ActivityReport.findAll({
+    where: {
+      [Op.or]: [
+        {
+          status: 'submitted', approvingManagerId: userId,
+        },
+        {
+          [Op.and]: [
+            {
+              status: {
+                [Op.ne]: 'approved',
```

Comment:
Should approved reports be shown on the alerts? It seems like that would get pretty cluttered over time.

Reply:
True.  No need to have approved reports. Op.ne (not equal) will take care of it.

Reply:
Hmm...Op.ne (not equal) should take care of not bringing in approved reports. Do you still see 'approved' in the alerts section?

Reply:
![Screenshot from 2021-02-16 13-07-06](https://user-images.githubusercontent.com/6248903/108103498-f0531180-7081-11eb-970a-519d6fb98228.png)

This report was filled out and submitted by userid 5 and approved by userid 1. When I log backin as userid 5 it shows up.